pkg/authorization/authorizer/authorizer.go

```diff
@@ -509,21 +521,42 @@ func GetBootstrapPolicyBinding(masterNamespace string) *authorizationapi.PolicyB
 		LastModified: util.Now(),
 		PolicyRef:    kapi.ObjectReference{Namespace: masterNamespace},
 		RoleBindings: map[string]authorizationapi.RoleBinding{
-			"Components": {
+			"system:components": {
 				ObjectMeta: kapi.ObjectMeta{
-					Name:      "Components",
+					Name:      "system:components",
 					Namespace: masterNamespace,
 				},
 				RoleRef: kapi.ObjectReference{
-					Name:      "ComponentRole",
+					Name:      "system:components",
+					Namespace: masterNamespace,
+				},
+				UserNames: []string{"system:openshift-client", "system:kube-client"},
```

Comment:
> Should probably give system:openshift-client access to all openshift resources, and system:kube-client access to all kubernetes resources.

I'd like to make that change post-resource groups.  Otherwise it's impossible to read.